scripts/config/msu/text.nut

```diff
@@ -13,4 +13,23 @@
 	{
 		return this.color(::Const.UI.Color.NegativeValue, _string);
 	}
+
+	function colorizeValue( _value, _invert = false, _addSign = true )
+	{
+		if (_value < 0) return this[_invert ? "colorGreen" : "colorRed"](_value);
+
+		if (_value > 0) return (_addSign ? "+" : "") + this[_invert ? "colorRed" : "colorGreen"](_value);
+
+		if (_value == 0) return this[_invert ? "colorRed" : "colorGreen"](_value);
+	}
+
+	function getIncreaseDecrease( _value, _invert = false )
+	{
+		return (_value >= 0 || _invert) ? "increase" : "decrease";
+	}
+
+	function getIncreaseReduce( _value, _invert = false )
```

Comment:
I've added separate functions for `decrease` and `reduce` because I think both words fit differently in a sentence based on the context. However, we could remove one of these functions. What do you say?

Reply:
I think increase & decrease is fine unless you have a specific example where you'd wanna use increase/reduce in which case leave both.

Reply:
> specific example where you'd wanna use increase/reduce
Reduced example: Damage to Hitpoints is **reduced** by x%. In this case "decreased" doesn't sound as good.

Honestly I think "reduce" will be used much more than "decrease".

Reply:
Maybe a different function for this kinda thing would be better, allow the modder to pass their increase/decrease equivalent values?

Reply:
Sounds good, I will try to write something up for that.